frontend/src/pages/chat/libs/constants/default-input.constant.ts

```diff
@@ -0,0 +1,8 @@
+const DEFAULT_INPUT = {
+  defaultValues: {
+    text: '',
+  },
+  mode: 'onSubmit',
+} as const;
```

Comment:
we need to store only defaultValues in this constant
```suggestion
const CHAT_INPUT_DEFAULT_VALUES = {
    text: '',
} as const;
```